src/models/TerraformComponentAttributeDefinition.js

```diff
@@ -0,0 +1,22 @@
+import { ComponentAttributeDefinition } from 'leto-modelizer-plugin-core';
+
+class TerraformComponentAttributeDefinition extends ComponentAttributeDefinition {
+  /**
+   * Override ComponentAttribute constructor with isDynamic property.
```

Comment:
you talk about isDynamic property but not present in the code. Please update documentation

Reply:
Done